skimage/metrics/simple_metrics.py

```diff
@@ -240,15 +244,15 @@ def normalized_mutual_information(image0, image1, *, bins=100):
     References
     ----------
     .. [1] C. Studholme, D.L.G. Hill, & D.J. Hawkes (1999). An overlap
-           invariant entropy measure of 3D medical image alignment.
-           Pattern Recognition 32(1):71-86
-           :DOI:`10.1016/S0031-3203(98)00091-0`
+            invariant entropy measure of 3D medical image alignment.
```

Comment:
When did the syntax/convention change? Just curious. :pray:

Reply:
Hello, Thank for the modifications. I am not totally sure which syntax/convention you are referring to. Is it in the docstring? I propose a change in simple_metrics which is a bit a side effect of the main proposal. This is to have a uniform support of weights in the registration code across the two implemented methods. I hope this is backward compatible and doesn't break anything.

Reply:
I'm referring to the extra indent, which I also noticed lines 18-19 of `doc/examples/registration/plot_register_affine.py`. Perhaps the formatting got fixed/taken care of by the pre-commit hook, in which case you didn't actually *decide* on it. I was just curious about the change because, up until now(?), we would write:
```rst
.. [1] Juan Nunez-Iglesias, Stefan van der Walt, and Harriet Dashnow. Elegant
       SciPy: The Art of Scientific Python. 1st. O'Reilly Media, Inc.,
       2017. isbn: 1491922877, 9781491922873.
```
and not
```rst
.. [1] Juan Nunez-Iglesias, Stefan van der Walt, and Harriet Dashnow. Elegant
        SciPy: The Art of Scientific Python. 1st. O'Reilly Media, Inc.,
        2017. isbn: 1491922877, 9781491922873.
```